CorsixTH/Lua/hospital.lua

```diff
@@ -2456,3 +2465,7 @@ end
 --! Called when the vip is out of the hospital grounds
 function Hospital:makeVipEndFax(vip_rating, name, cash_reward, vip_message)
 end
+
+--! Give the player information with the UIInformation dialog
+function Hospital:giveInfo(message)
+end
```

Comment:
`giveInfo` is highly generic, while the description is very specific.

For example, the function name doesn't exclude using speech to "give information".
The description is more about "how" rather  than "what does the function aim to do".

As for changing this, my guess is to avoid "how" (basically, the way to get such detailed information should be to read the actual code or comments in that code), and just tell the aim. Not sure if you want to exclude eg speech or other ways to give information to the player. In general there is nothing wrong with keeping possibilities that we currently don't have open. Who knows what the future will bring.